plugins/woocommerce/src/Internal/DataStores/Orders/CustomOrdersTableController.php

```diff
@@ -479,14 +479,13 @@ private function process_pre_update_option( $option, $old_value, $value ) {
 			return $value;
 		}
 
-		// TODO: Re-enable the following code once the COT to posts table sync is implemented (it's currently disabled to ease testing).
-
-		/*
+		/**
+		 * Commenting out for better testability.
 		$sync_is_pending = 0 !== $this->data_synchronizer->get_current_orders_pending_sync_count();
```

Comment:
Shouldn't we restore this code to prevent changing datastores while sync hasn't completed? 😬

Reply:
actually, this was already commented out (and should stay that way for some more time). i just changed the comment style, and added the reason for commenting out.

Reply:
Got it!